aten/src/ATen/native/native_functions.yaml

```diff
@@ -3606,13 +3606,13 @@
   variants: method
   dispatch:
     CPU: pow_
-    CUDA: legacy::cuda::_th_pow_
+    CUDA: pow_
 
 - func: pow_.Tensor(Tensor(a!) self, Tensor exponent) -> Tensor(a!)
   variants: method
   dispatch:
     CPU: pow_
-    CUDA: legacy::cuda::_th_pow_
+    CUDA: pow_
```

Comment:
You can remove the `dispatch:` for this, as well as a lot of other call in here.